sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -270,12 +269,15 @@ object ScalaJSPluginInternal {
 
   private def scalajspSettings: Seq[Setting[_]] = {
     case class Options(
-        infos: Boolean = false
+        infos: Boolean = false,
+        raw: Boolean = false
     )
 
     val optionsParser: Parser[Options] = {
       token(OptSpace ~> (
-          (literal("-i") | "--infos") ^^^ ((_: Options).copy(infos = true))
+          (literal("-i") | "--infos") ^^^ ((_: Options).copy(infos = true)) |
+            (literal("-r") | "--raw") ^^^ ((_: Options).copy(raw = true))
+
```

Comment:
For parser combinators, prefer this style:
```scala
      token(OptSpace ~> (
          (literal("-i") | "--infos") ^^^ ((_: Options).copy(infos = true))
        | (literal("-r") | "--raw") ^^^ ((_: Options).copy(raw = true))
      )
```
(you also have a spurious blank line, here)